pymeasure/instruments/rohdeschwarz/fsl.py

```diff
@@ -290,7 +290,7 @@ def delete_channel(self, channel_name):
         Args:
             channel_name (string): name of the channel to be deleted
         """
-        channels = self.show_channels
+        channels = self.available_channels
```

Comment:
No need to define a local variable here, simply use `self.available_channel` in the following line. The instrument is only called once.